rcl/src/rcl/timer.c

```diff
@@ -265,10 +265,18 @@ rcl_timer_clock(rcl_timer_t * timer, rcl_clock_t ** clock)
 
 rcl_ret_t
 rcl_timer_call(rcl_timer_t * timer)
+{
+  rcl_timer_call_info_t info;
+  return rcl_timer_call_with_info(timer, &info);
+}
+
+rcl_ret_t
+rcl_timer_call_with_info(rcl_timer_t * timer, rcl_timer_call_info_t * call_info)
```

Comment:
test would be ideal to align with `rcl_timer_call` to make sure it returns `rcl_timer_call_info_t`.